components/match2/commons/starcraft_starcraft2/match_group_input_starcraft.lua

```diff
@@ -95,9 +95,15 @@ end
 
 function StarCraftMatchGroupInput.checkFinished(match)
 	local lang = match.lang
-	if Logic.readBool(match.finished) or (not Logic.isEmpty(match.winner)) then
+
+	if match.finished == 'false' then
+		match.finished = false
+	elseif not Logic.isEmpty(match.winner) then
 		match.finished = true
-	else
+	end
+
+	-- match.finished is automatically set after about a day, in case results are not entered after the match
```

Comment:
```suggestion
	-- match is automatically set after after a certain amount of time
	-- (depending on the fact if the date is exact or not)
```
in case of an exact date it is ~8.6h and not a day